helm/botkube/values.yaml

```diff
@@ -116,25 +118,25 @@ sources:
     # -- Describes Kubernetes source configuration.
     # @default -- See the `values.yaml` file for full object.
     botkube/kubernetes:
-      context: &defaultPluginContext
+      context: &default-plugin-context
         defaultNamespace: "default"
         # -- RBAC configuration for this plugin.
         rbac:
           # -- Static impersonation for a given username and groups.
-          group:
+          user:
```

Comment:
Moved `user` before `group`. I find it more readable this way.

Reply:
Let's comment out the user mapping completely from the default configuration.

So the default configuration will be only group mapping (`botkube-plugins-read-only`) and that's it. And we'll use the "internal" user for impersonation.